tests/common.py

```diff
@@ -143,6 +143,23 @@ def reset(self, seed=None, options=None):
         return self.observation_space.sample(), {}
 
 
+class DummyTextEnv(gym.Env):
+    def __init__(self):
+        self.action_space = spaces.Text(max_length=10, min_length=2, charset="01")
```

Comment:
I think it would be good to create a dummy env with a full unicode charset minus surrogate chars that returns emoji as observations, as a stress test.

Reply:
This is the case for `DummyTextEnv` for observation, isn't it? 
Do you want it also for action space?